python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -274,6 +294,105 @@ def plot_one_image(ax, data, name):
+        psf_color = "red"
+    else:
+        psf_color = text_color
+    fig.text(0.0, 0.87, "PSF (nJy):", color=psf_color)
+    fig.text(0.25, 0.87, f"{source['psFlux']:8.1f}", horizontalalignment='right')
+    fig.text(0.252, 0.87, "+/-", color=text_color)
+    fig.text(0.29, 0.87, f"{source['psFluxErr']:8.1f}")
+    fig.text(0.40, 0.87, "S/N:", color=text_color)
+    fig.text(0.45, 0.87, f"{abs(source['psFlux']/source['psFluxErr']):6.2f}")
+
+    # NOTE: yellow is hard to read on white; use goldenrod instead.
+    if source['flags'] & 2:
+        fig.text(0.55, 0.87, "EDGE", color="goldenrod", fontweight="bold")
+    # combine interpolated and interpolatedCenter flags
+    if source['flags'] & 3 or source['flags'] & 8:
+        fig.text(0.62, 0.87, "INTERP", color="green", fontweight="bold")
+    # combine saturated and saturatedCenter flags
+    if source['flags'] & 4 or source['flags'] & 9:
+        fig.text(0.72, 0.87, "SAT", color="green", fontweight="bold")
+    # combine cosmicray and cosmicrayCenter flags
+    if source['flags'] & 5 or source['flags'] & 10:
+        fig.text(0.77, 0.87, "CR", color="magenta", fontweight="bold")
+    if source['flags'] & 6:
+        fig.text(0.81, 0.87, "BAD", color="red", fontweight="bold")
+    if source['isDipole']:
+        fig.text(0.87, 0.87, "DIPOLE", color="indigo", fontweight="bold")
+
+    # color the aperture flux label if any aperture flag is set
+    if source['flags'] & 15 or source['flags'] & 16:
+        ap_color = "red"
+    else:
+        ap_color = text_color
+    fig.text(0.0, 0.83, "ap (nJy):", color=ap_color)
+    fig.text(0.25, 0.83, f"{source['apFlux']:8.1f}", horizontalalignment='right')
+    fig.text(0.252, 0.83, "+/-", color=text_color)
+    fig.text(0.29, 0.83, f"{source['apFluxErr']:8.1f}")
+    fig.text(0.40, 0.83, "S/N:", color=text_color)
+    fig.text(0.45, 0.83, f"{abs(source['apFlux']/source['apFluxErr']):#6.2f}")
+
+    # combine suspect and suspectCenter flags
+    if source['flags'] & 7 or source['flags'] & 11:
+        fig.text(0.55, 0.83, "SUS", color="goldenrod", fontweight="bold")
+    if source['flags'] & 12:
+        fig.text(0.60, 0.83, "CENTROID", color="red", fontweight="bold")
+    if source['flags'] & 13:
+        fig.text(0.73, 0.83, "CEN+", color="chocolate", fontweight="bold")
+    if source['flags'] & 14:
+        fig.text(0.80, 0.83, "CEN-", color="blue", fontweight="bold")
+    if source['flags'] & 23 or source['flags'] & 24 or source['flags'] & 25 \
+       or source['flags'] & 26 or source['flags'] & 27:
+        fig.text(0.87, 0.83, "SHAPE", color="red", fontweight="bold")
+
+    # color the forced flux label if any forced flag is set
+    if source['flags'] & 20 or source['flags'] & 21 or source['flags'] & 22:
+        total_color = "red"
+    else:
+        total_color = text_color
+    fig.text(0.0, 0.79, "total (nJy):", color=total_color)
+    fig.text(0.25, 0.79, f"{source['totFlux']:8.1f}", horizontalalignment='right')
```

Comment:
My preference would be for the same font color/weight in both kay and value; it seems like the values are coming in with a heavier weight, or maybe it's just a different color.

Reply:
I experimented with various choices, and I find it much easier to pick out the numbers when the labels are less obvious. The weights are the same, it's just black vs. grey for the text color.